apps/examples/src/15-custom-zones/ZonesExample.tsx

```diff
@@ -12,7 +12,7 @@ function CustomShareZone() {
 	return (
 		<div
 			style={{
-				backgroundColor: 'var(--palette-light-blue)',
+				backgroundColor: DefaultColorThemePalette.lightMode['light-blue'].solid,
```

Comment:
This should use some other ui color, like `tl-selected`. It should not have originally referenced the shape color palette.